<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TouristBookMainWindowClass</class>
 <widget class="QMainWindow" name="TouristBookMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Справочник туриста</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Браузер:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="BrowserView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Время последнего автосохранения:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="autoSaveTime">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="displayFormat">
           <string>hh:mm:ss</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="CreateButton">
          <property name="text">
           <string>Создать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteButton">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="RecordBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>251</width>
          <height>181</height>
         </size>
        </property>
        <property name="title">
         <string>Текущая запись:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Вид отдыха: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="KindOfRecreationBox">
           <item>
            <property name="text">
             <string>пляжный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>экскурсионный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>экзотический</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>круиз</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>VIP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>лечебный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>экстремальный</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>горнолыжный</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Страна: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="CountryBox">
           <item>
            <property name="text">
             <string>Россия</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Абхазия</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Турция</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Египет</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Место отдыха: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="RecreationPlaceEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>50</number>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Стоимость путевки: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="TicketCost">
           <property name="minimum">
            <number>3000</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Длительность: </string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="Duration">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="VisaCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Требуется виза</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>CountryBox</zorder>
        <zorder>label_4</zorder>
        <zorder>RecreationPlaceEdit</zorder>
        <zorder>label_5</zorder>
        <zorder>TicketCost</zorder>
        <zorder>label_6</zorder>
        <zorder>Duration</zorder>
        <zorder>VisaCheckBox</zorder>
        <zorder>label_2</zorder>
        <zorder>KindOfRecreationBox</zorder>
        <zorder>label_3</zorder>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="OpenFile">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="NewDB">
   <property name="text">
    <string>Новая БД</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="SaveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>CreateButton</tabstop>
  <tabstop>DeleteButton</tabstop>
  <tabstop>BrowserView</tabstop>
  <tabstop>KindOfRecreationBox</tabstop>
  <tabstop>CountryBox</tabstop>
  <tabstop>RecreationPlaceEdit</tabstop>
  <tabstop>TicketCost</tabstop>
  <tabstop>Duration</tabstop>
  <tabstop>VisaCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="touristbookmainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
